cli/index.ts

```diff
@@ -159,8 +161,16 @@ YargsParser.command(
 async function serve(port: number, pathToSpec: string, options: Options = {}) {
   let spec = await loadAndBundleSpec(pathToSpec);
   let pageHTML = await getPageHTML(spec, pathToSpec, options);
+  let isUrl;
+  try {
+    // tslint:disable-next-line
+    new URL(pathToSpec);
```

Comment:
This is not a valid URL check, e.g. `./some/folder/openapi.js` is a valid relative URL. You have to check if this is an absolute URL. I would keep it simple, just check if `pathToSpec` starts with `http://` or `https://`